dask/dataframe/io.py

```diff
@@ -338,8 +338,11 @@ def from_pandas(data, npartitions, sort=True):
     ----------
     df : pandas.DataFrame or pandas.Series
         The DataFrame/Series with which to construct a dask DataFrame/Series
-    npartitions : int
+    npartitions : int, default None
```

Comment:
No problem. 

The reason I went that route was that you have to pick one... so you can't option for neither. But I'll go with it. There's a check that you only have picked one anyways.